### DISCUSSION TITLE
proposal: noSvgWithoutTitle should allow for 'svgComponents' to tell the linter which components are SVGs

### DISCUSSION THREAD

**Discussion by lifeiscontent** [biomejs/biome#3890] [Ideas]
### Environment information

```block
Currently there's no way to continue to enforce linter rules for noSvgWithoutTitle for components, it'd be nice to have a `svgComponents` option similar to `noLabelWithoutControl` with `labelComponents`
```

### What happened?

https://biomejs.dev/playground/?code=ZgB1AG4AYwB0AGkAbwBuACAAUwB0AGEAdAB1AHMAKABwAHIAbwBwAHMAOgAgAFIAZQBhAGMAdAAuAEMAbwBtAHAAbwBuAGUAbgB0AFAAcgBvAHAAcwBXAGkAdABoAG8AdQB0AFIAZQBmADwAJwBzAHYAZwAnAD4AKQAgAHsACgAgACAAcgBlAHQAdQByAG4AIAAoAAoAIAAgACAAIAAJAAkAPABzAHYAZwAKACAAIAAgACAAIAAgACAAIAAgACAAIAAgACAAIAB7AC4ALgAuAHAAcgBvAHAAcwB9AAoACQAJAAkAdwBpAGQAdABoAD0AIgA5ACIACgAJAAkACQBoAGUAaQBnAGgAdAA9ACIAOQAiAAoACQAJAAkAdgBpAGUAdwBCAG8AeAA9ACIAMAAgADAAIAA5ACAAOQAiAAoACQAJAAkAZgBpAGwAbAA9ACIAbgBvAG4AZQAiAAoACQAJAAkAeABtAGwAbgBzAD0AIgBoAHQAdABwADoALwAvAHcAdwB3AC4AdwAzAC4AbwByAGcALwAyADAAMAAwAC8AcwB2AGcAIgAKAAkACQA%2BAAoACQAJAAkAPABjAGkAcgBjAGwAZQAgAGMAeAA9ACIANAAuADUAIgAgAGMAeQA9ACIANAAuADUAIgAgAHIAPQAiADMALgA3ADUAIgAgAHMAdAByAG8AawBlAD0AIgB3AGgAaQB0AGUAIgAgAHMAdAByAG8AawBlAFcAaQBkAHQAaAA9ACIAMQAuADUAIgAgAC8APgAKAAkACQA8AC8AcwB2AGcAPgAKACAAIAApAAoAfQAKAAoACgBmAHUAbgBjAHQAaQBvAG4AIABFAHgAYQBtAHAAbABlACgAKQAgAHsACgAgACAAcgBlAHQAdQByAG4AIAA8AFMAdABhAHQAdQBzACAALwA%2BADsACgB9AA%3D%3D

### Expected result

I'd expect to be able to define an `svgComponents` in Biome

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Moved to a discussion because it's not a bug.

Can you please update the description, and provide a good proposal for what you're asking?
